ops/ccs-ops-misc/load_test/.vscode/settings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "files.insertFinalNewline": true,
+  "python.formatting.provider": "black",
+  "[python]": {
+    "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
+      "source.organizeImports": true
+    }
+  },
+  "python.sortImports.args": ["--profile", "black"],
+  "python.linting.pylintEnabled": true,
+  "python.linting.enabled": true,
+  // There appears to be issues with Pylint+VSCode resolving module paths for local
+  // directories, especially when using virutal environments. This is not
+  // disabled in .pylintrc, so running pylint manually will check imports.
+  // See https://github.com/Microsoft/vscode-python/issues/3840 and
+  // https://github.com/Microsoft/python-language-server/issues/887.
+  // None of the proposed solutions seemed to work, other than disabling this
+  // linting error
+  "python.linting.pylintArgs": ["--disable=import-error"]
```

Comment:
I might have an idea for this, but it'll be out of band of this PR. Basically, I want to try defining a `.vscode-workspace` file _and_ a dev container for the workspace.

Reply:
Sounds good to me, I feel like we can definitely go down that avenue at some point in the future (maybe whenever we decide to include pre-commit hooks or move this into its own repo?).